factories/__init__.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```

Comment:
Greg said not to use PEP8 in the `models.py` files. Please, add this line to those files as well.

Reply:
Not yet, I'm investigating how to not apply flake8 for a certain rule, not cancel it completely. I will make it in a separate PR.

Reply:
ok